src/assets/i18n/en.json5

```diff
@@ -1640,6 +1640,10 @@
 
   "cookies.consent.app.description.authentication": "Required for signing you in",
 
+  "cookies.consent.app.title.correlation-id": "Correlation ID",
+
+  "cookies.consent.app.description.correlation-id": "Allow us to track your session for debugging purposes",
```

Comment:
I'd recommend a slightly difference wording for this text in order to make it clear that this is for backend logs.  Maybe something like this:
```
"Allow us to track your session in backend logs for support/debugging purposes"
```